packages/cli/fern-definition/ir-to-jsonschema/src/__test__/__snapshots__/examples/type_types_ExtendedMovie.json

```diff
@@ -12,22 +13,29 @@
         {
           "type": "null"
         }
-      ]
+      ],
+      "title": "prequel"
     },
     "title": {
-      "type": "string"
+      "type": "string",
+      "title": "title"
     },
     "from": {
-      "type": "string"
+      "type": "string",
+      "title": "from"
     },
     "rating": {
-      "type": "number"
+      "type": "number",
+      "title": "rating",
```

Comment:
is there value in having a title here?

Reply:
@dsinghvi I looked into how other popular schemas / schema tools handle titles (e.g. VSCode settings.json Intellisense), and I now agree that there's little to no value in having the property key in the title.

Regarding using the type name instead: since JSONSchema suggests that `title` and `description` should document **purpose**, I think there's also little value from doing that, since the name of a type might not provide any useful info in a doc comment that the schema validation doesn't already provide (I can provide more detail on this if you'd like). But its possible for us to put something like `type: FooSchema` at the top of the doc comment, but I'm indifferent about it since I don't see other schema tools adding that info in.

Reply:
Removed `title` generation.